tests/test_dsp_decibel.py

```diff
@@ -58,3 +58,31 @@ def test_return_prefix():
     test_signal = pf.Signal([0, 1, 0], sampling_rate=44100, fft_norm='power')
     dB_sig, prefix = pf.dsp.decibel(test_signal, return_prefix=True)
     assert prefix == 10
+
+
+def test_decibel_complex():
+    test_signal = [0.01, 0.1, 1, 10, 100]
+    test_Signal = pf.Signal(test_signal, 44100, is_complex=True)
+    # Test right calculation
+    # and default values log_prefix = 20 & log_reference = 1
+    npt.assert_equal(pf.dsp.decibel(test_Signal, domain='time'),
+                     [[-40, -20, 0, 20, 40]])
+    npt.assert_equal(pf.dsp.decibel(test_Signal, domain='freq'),
+                     20*np.log10(np.abs(test_Signal.freq)))
+    npt.assert_equal(pf.dsp.decibel(test_Signal, domain='freq_raw'),
+                     20*np.log10(np.abs(test_Signal.freq_raw)))
+    # Testing overwrite log_prefix Parameter
```

Comment:
It might be possible to remove all tests below, since they are already tested above

Reply:
But not for complex signals, or do I miss something here?

Reply:
I thought it might not be necessary to test all options of the function with complex signals. For example it should be enough testing the log_prefix with one signal. It's also fine to have it in there, but it might save time for further pulls to have an eye on this.